recipes/py_gd/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.1.6" %}
+{% set libgd_version = "2.2.2" %}
+
+package:
+    name: py_gd
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/NOAA-ORR-ERD/py_gd.git
+    git_tag: v{{ version }}
```

Comment:
Does GitHub provide tarballs?

I really would rather get a pypi package up, but haven't had s chance yet.

-CHB

Sent from my iPhone

On Jul 17, 2016, at 3:05 PM, jakirkham <notifications@github.com> wrote:

In recipes/py_gd/meta.yaml
<https://github.com/conda-forge/staged-recipes/pull/1086#discussion_r71090461>
:

> @@ -0,0 +1,47 @@
> +{% set version = "0.1.6" %}
> +{% set libgd_version = "2.2.2" %}
> +
> +package:
> +    name: py_gd
> +    version: {{ version }}
> +
> +source:
> +    git_url: https://github.com/NOAA-ORR-ERD/py_gd.git
> +    git_tag: v{{ version }}

We prefer to use tarballs over cloning from version control. So please
change like so. Please fill in the checksum.

fn: py_gd-{{ version }}.tar.gz
url: https://github.com/NOAA-ORR-ERD/py_gd/archive/v{{ version }}.tar.gz
sha256: ???

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/conda-forge/staged-recipes/pull/1086/files/e15a7c4670f156270cc51d224adf212e421c72d2#r71090461>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/AA38YNEGLhWL1lrtBc2K03gdZOFns-N7ks5qWqdEgaJpZM4JOTtB>
.